crates/rpc/rpc/src/eth/error.rs

```diff
@@ -527,7 +528,7 @@ pub enum RpcPoolError {
     #[error(transparent)]
     Invalid(#[from] RpcInvalidTransactionError),
     /// Custom pool error
-    #[error("{0:?}")]
+    #[error("{0}")]
```

Comment:
transparent here?

Reply:
Needs `impl std::error::Error for Box<dyn _>`